tmt/cli.py

```diff
@@ -600,6 +601,25 @@ def export(context, format_, nitrate, bugzilla, **kwargs):
             f"Invalid test export format '{format_}'.")
 
 
+@tests.command()
+@click.pass_context
+@name_filter_condition
+@verbose_debug_quiet
+@force_dry
+def uuid(context, **kwargs):
+    """
+    generate UUIDs to test  leafs if not already defined.
```

Comment:
double space, maybe some capitalisation would look better in --help

Reply:
capitalizaion it is question, we'll see, I have no strong opinion. Let's discuss with others.

Reply:
Yes, let's use `Generate...` to make it consistent with the rest of options.

Reply:
Will cover this.